File: fyp/server/controller/prediction model/predict.py

import pickle
import sys

inputt=[[]]
for v in sys.argv[1:]:
  inputt[0].append(int(v))



x= inputt


model = pickle.load(open('D:/FYP/fyp/server/controller/prediction model/model.sav', 'rb'))
y=model.predict(x)

a=int(y)
b=str(a)
c=b[0:len(b)-2]+'00'
c=c.replace('-','')
print(c)

    

File: fyp/server/controller/predict.js
var {PythonShell} = require('python-shell');


exports.predict=async (req,res)=>{
  console.log(req.body.inputt)
  try{
    var options = {
      
      pythonOptions: ['-u'],
      args: req.body.inputt
    };
    console.log("ok")

    PythonShell.run('D:/FYP/fyp/server/controller/prediction model/predict.py', options, function (err, results) {
      if (err) 
        throw err;
      
      console.log(results[0]);
      res.json({price:results[0],success:true})
    });
}
catch(err){
  res.json({ message: err.message,success:false }); 
}

}